201-encrypt-running-windows-vm/azuredeploy.json

```diff
@@ -26,120 +14,137 @@
         "description": "Name of the KeyVault to place the volume encryption key"
       }
     },
-    "keyVaultResourceGroup": {
-      "type": "string",
-      "metadata": {
-        "description": "Resource group of the KeyVault"
-      }
-    },
-    "useExistingKek": {
-      "type": "string",
-      "defaultValue": "nokek",
-      "allowedValues": [
-        "nokek",
-        "kek"
-      ],
-      "metadata": {
-        "description": "Select kek if the secret should be encrypted with a key encryption key and pass explicit keyEncryptionKeyURL. For nokek, you can keep keyEncryptionKeyURL empty."
-      }
-    },
-    "keyEncryptionKeyURL": {
+    "keyVaultDomain": {
       "type": "string",
-      "defaultValue": "",
       "metadata": {
-        "description": "URL of the KeyEncryptionKey used to encrypt the volume encryption key"
+        "description": "Domain of the KeyVault to place the volume encryption key"
       }
     },
-    "volumeType": {
-      "type": "string",
-      "defaultValue": "All",
-      "metadata": {
-        "description": "Type of the volume OS or Data to perform encryption operation"
-      }
-    },
-    "sequenceVersion": {
-      "type": "string",
-      "defaultValue": "1.0",
-      "metadata": {
-        "description": "Pass in an unique value like a GUID everytime the operation needs to be force run"
-      }
-    },
-    "location": {
-      "type": "string",
-      "defaultValue": "[resourceGroup().location]",
-      "metadata": {
-        "description": "Location for all resources."
+      "enabledForDeployment": {
+        "type": "bool",
+        "defaultValue": true,
+        "allowedValues": [
+          true,
```

Comment:
remove allowedValues (since this is a bool) - remove for the other bools below too

Reply:
done